src/api/spec/controllers/webui/image_templates/interconnects_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+require './spec/support/shared_examples/controllers/image_templates.rb'
+
+RSpec.describe Webui::ImageTemplates::InterconnectsController, type: :controller do
+  describe 'GET #index' do
+    it_behaves_like 'image templates', 'interconnects/index', :xml
+
+    context 'and format HTML' do
+      it 'fails with UnknownFormat' do
+        expect do
+          get :index, format: :html
+        end.to raise_error ActionController::UnknownFormat
```

Comment:
I'm not sure if I think this is a good idea? Should we be testing what something is not? We've already specified that it behaves like image templates using xml format so why should we say that it doesn't accept html format? 

What do you think?